source/MainWindow.cpp

```diff
@@ -95,6 +96,7 @@ MainWindow::MainWindow(BRect frame, BEntry* assocEntry, SettingsFile* settings)
 	fSettingsItem = new BMenuItem("Settings", new BMessage(MSG_SETTINGS));
 
 	fHelpMenu = new BMenu("Help", B_ITEMS_IN_COLUMN);
+	fAboutItem = new BMenuItem("About", new BMessage(B_ABOUT_REQUESTED));
```

Comment:
There doesn't seem a standard for this, but I think it's nicer to have "About ResourceEdit" for the label.